app/views/questions/edit.html.erb

```diff
@@ -1,6 +1,8 @@
-<h1>Editing Question</h1>
+<body class="container">
```

Comment:
I think this might make you have a body tag inside of an additional body tag.  You are only allowed one `body` tag per page.  

That being said you can apply bootstrap classes to basically any tag, `section`, `div`, `main` are all valid here.